test-setup.js

```diff
@@ -37,6 +37,10 @@ const fetch = jest.fn(
 );
 global.fetch = fetch;
 global.Headers = Headers;
+global.localStorage = {};
+global.localStorage.setItem = (key, value) => {
+	global.localStorage[key] = value;
+};
```

Comment:
```javascript
global.localStorage = {
	setItem(key, value) {
		global.localStorage[key] = value;
	}
}
```

But, is it really necessary to mock that ? Perhaps the use of localStorage + `afterEach` to clear it will do the trick